source/server/listener_manager_impl.h

```diff
@@ -152,6 +152,12 @@ class DrainingFilterChainsManager {
     drain_timer_->enableTimer(drain_time);
   }
 
+  void addFilterChainToDrain(const Network::FilterChain* filter_chain) {
```

Comment:
nit: pass by const reference since can't be null

Reply:
Nice catch!